torch/nn/quantized/modules/linear.py

```diff
@@ -119,7 +119,7 @@ def __init__(self, in_features, out_features, bias_=True):
         qweight = torch._empty_affine_quantized(
             [out_features, in_features], scale=1, zero_point=0, dtype=torch.qint8)
 
-        self.set_weight(qweight)
+        self.set_weight_bias(qweight, self.bias)
```

Comment:
so the problem is that you already quantized the bias above, right?

Reply:
Right: maybe use c10::optional<Tensor> bias to make bias optional.